.github/workflows/prepare-release.yaml

```
@@ -148,6 +148,7 @@ jobs:
           - [ ] Review the changes in [the release commit]($COMMIT_URL)
           - [ ] Review the [Draft Release](${{ needs.create-release.outputs.release-url }})
           - [ ] Review **and approve** this PR
+          - [ ] For release: corresponding [release notes](https://github.com/SAP/ai-sdk-java/blob/main/docs/release-notes/release-notes-0-to-14.md) copied to [GitHub Releases](https://github.com/SAP/ai-sdk-java/releases)  
```

```suggestion
          - [ ] Add release notes to the [Draft Release](${{ needs.create-release.outputs.release-url }}
          - [ ] Review **and approve** this PR
```
We already had a check for the draft release. Also draft releases are not in https://github.com/SAP/ai-sdk-java/releases

---

Ok, maybe some misunderstanding of the process from my side. So *Draft Release* is the part that after release is shown in https://github.com/SAP/ai-sdk-java/releases?

---

When the draft is published (after the perform release), it is shown in the link.